.eslintrc

```diff
@@ -74,6 +73,13 @@
         ]
       }
     ],
+    "@typescript-eslint/ban-types": [
```

Comment:
I started replacing `Function` types with the proper ones, however, there are too many of them and it's supposed to be a small PR. Must be resolved later on.